services/pg-backup/Makefile

```diff
@@ -26,15 +26,22 @@ up-master: up-dalco
 
 .PHONY: up-public ## Deploys pg-backup stack for public access cluster
 up-public: .init ${TEMP_COMPOSE}-public
-	docker stack deploy --with-registry-auth --prune --compose-file ${TEMP_COMPOSE}-public ${STACK_NAME}
-
+	docker stack deploy --with-registry-auth --prune --compose-file ${TEMP_COMPOSE}-public ${STACK_NAME}; \
 
 # Helpers -------------------------------------------------
 
 .PHONY: ${TEMP_COMPOSE}
 ${TEMP_COMPOSE}: docker-compose.yml
-	@docker-compose -f $< --log-level=ERROR config > $@
+	docker-compose -f $< --log-level=ERROR config > $@
+	set -o allexport; \
+	source $(REPO_CONFIG_LOCATION); \
+	sed -i "s/SERVICE_NAME_PARAMETRIZED/$${PREFIX_STACK_NAME}_pg_backup/g" $@;\
+	set +o allexport; \
 
 .PHONY: ${TEMP_COMPOSE}-public
 ${TEMP_COMPOSE}-public: docker-compose.yml docker-compose.public.yml
 	docker-compose -f $< -f docker-compose.public.yml --log-level=ERROR config > $@
+	set -o allexport; \
+	source $(REPO_CONFIG_LOCATION); \
+	sed -i "s/SERVICE_NAME_PARAMETRIZED/$${PREFIX_STACK_NAME}_pg_backup/g" $@;\
```

Comment:
why not use jinja2 here? Much more modern and powerful and less error prone. 

If you need an example on how to generate a valid docker yml file from a j2 template, see:
https://github.com/ITISFoundation/osparc-ops-environments/blob/14569774c02307fc508fceb5e967337c9a924107/services/traefik/Makefile#L104

Reply:
Done